clients/js/src/Collection.ts

```diff
@@ -240,6 +240,7 @@ export class Collection {
             },
             this.api.options
         )
+            .then(() => true)
```

Comment:
Is there no way to handle this using the `handleSuccess` method? If the actual upsert does not return true, then this is a problem with the API, not so much the client.

Reply:
Yeah the upsert and update methods in the /server/fastapi/_init_.py return None. The documentation on the python client states that the return type for upsert and update are None. Do you guys want both methods to return true for all clients?

Reply:
I think we should be consistent across clients here